java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -1120,9 +1120,8 @@ private static Object[] geospatialCommands(BaseTransaction<?> transaction) {
                                         }
                                     },
                                 }, // geosearch(geoKey1, (15,37), BYBOX(400,400,km), ASC, COUNT 2)
-                                2L, // geosearch(geoKey2, geoKey1, (15,37), BYBOX(400,400,km), ASC, COUNT 2)
-                                2L, // geosearch(geoKey2, geoKey1, (15,37), BYBOX(400,400,km), STOREDIST, ASC, COUNT
-                                // 2)
+                                2L, // geosearchstore(geoKey2, geoKey1, (15,37), (400,400,km), ASC, 2)
+                                2L, // geosearchstore(geoKey2, geoKey1, (15,37), (400,400,km), STOREDIST, ASC, 2)
```

Comment:
nit: not sure if we care about this or not
```suggestion
                                2L, // geosearchstore(geoKey2, geoKey1, (15, 37), (400, 400, km), ASC, 2)
                                2L, // geosearchstore(geoKey2, geoKey1, (15, 37), (400, 400, km), STOREDIST, ASC, 2)
```